packages/gatsby-theme-store/src/sdk/orderForm/Provider.tsx

```diff
@@ -1,94 +1,75 @@
-/**
- * This provider starts fetching early and adds a suspendable reader to
- * the context. Use the hooks to interact with the context
- */
-import React, { createContext, useCallback, useEffect, useState } from 'react'
+import React from 'react'
 import type { FC } from 'react'
-
-import type { OrderFormFragment_OrderFormFragment } from './controller/__generated__/OrderFormFragment_orderForm.graphql'
-import type { OrderFormItem } from './types'
-
-const controller = () => import('./controller')
-
-export type OrderFormContext = {
-  value: OrderFormFragment_OrderFormFragment | null
-  addToCart: (items: OrderFormItem[]) => Promise<void>
-  updateItems: (items: OrderFormItem[]) => Promise<void>
+import {
+  OrderQueueProvider,
+  createOrderFormProvider,
+  DEFAULT_ORDER_FORM,
+  useOrderQueue,
+  useQueueStatus,
+} from '@vtex/order-manager'
+import { createOrderItemsProvider } from '@vtex/order-items'
+
+import { useClearOrderFormMessages } from './useClearOrderFormMessage'
+import { useToast } from './useToast'
+import { useLogger } from './useLogger'
+import { useGetOrderForm } from './useGetOrderForm'
+import { useMutateAddItems } from './useMutateAddItems'
+import { useMutateUpdateQuantity } from './useMutateUpdateQuantity'
+import type { OrderFormFragment_OrderFormFragment as OrderForm } from './controller/__generated__/OrderFormFragment_orderForm.graphql'
+
+const defaultOrderForm: OrderForm = {
```

Comment:
this could harm performance. We should measure this and see the JSON.parse approach

Reply:
Do you have a suggestion on how to fix it?

Reply:
I think he means this, right?
https://itnext.io/can-json-parse-be-performance-improvement-ba1069951839

Reply:
But it only enhances performance if use spread operator multiple times, right?